ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -1,5 +1,65 @@
 presubmits:
   openshift/openshift-azure:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e
+    decorate: true
+    labels:
+      artifacts: images
+    name: ci-openshift-openshift-azure-master-no-running-under-test
+    rerun_command: /test no-rut
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-secrets/azure
+        - --target=[output:stable:sync]
+        - --target=e2e-azure
```

Comment:
Note that this clashes with the original e2e job in the PR once this runs but I guess that's not a big issue. I don't know of a way to fix this.